### ISSUE TITLE
Text is rendered with visibly incorrect offsets

### ISSUE THREAD

**Issue by cart** [bevyengine/bevy#283]
This is clearly visible in the "scene" example. It is less visible for "smaller" text.
![image](https://user-images.githubusercontent.com/2694663/90946080-5d8d2f00-e3de-11ea-8ef8-1a390bc73caa.png)

**Comment by mockersf** [comment]
the scene example is now fixed with #725, but there's still issue of wavy test...

scene fixed:
before: <img width="300" alt="scene - b6004e44cff73876f2ba04f342bfe8129a7f5222" src="https://user-images.githubusercontent.com/8672791/97790339-197d8e00-1bc8-11eb-8d59-a2f4a51265c6.png"> - after: <img width="300" alt="scene - ef86ce98ed8a21eeb00e60c0b3bd7d4f17544c92" src="https://user-images.githubusercontent.com/8672791/97790340-1c787e80-1bc8-11eb-8b36-4c7a77d2c85d.png">

issue gets more noticeable with smaller text size (below 35 is more visible)
<details>
  <summary>Code to display text example</summary>

```rust
use bevy::prelude::*;

fn main() {
    App::build()
        .add_default_plugins()
        .add_startup_system(setup.system())
        .run();
}

fn setup(
    mut commands: Commands,
    asset_server: Res<AssetServer>,
    mut materials: ResMut<Assets<ColorMaterial>>,
) {
    commands
        // 2d camera
        .spawn(UiCameraComponents::default());

    commands.spawn(NodeComponents {
        style: Style {
            flex_direction: FlexDirection::Column,
            ..Default::default()
        },
        material: materials.add(Color::NONE.into()),
        ..Default::default()
    });

    commands.with_children(|container| {
        for i in 0..10 {
            let size = (20 + i * 5) as f32;
            container
            .spawn(TextComponents {
                style: Style {
                    size: Size {
                        height: Val::Px(size  + 10.),
                        ..Default::default()

                    },
                    ..Default::default()
                },
                text: Text {
                    value: format!("{}: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed non risus. Suspendisse lectus tortor, dignissim sit amet, adipiscing nec, ultricies sed, dolor.", size),
                    font: asset_server.load("fonts/FiraSans-Bold.ttf"),
                    style: TextStyle {
                        font_size: size ,
                        color: Color::WHITE,
                    },
                },
                ..Default::default()
            });
        }
    });
}
```
</details>

before: <img width="300" alt="b6004e44cff73876f2ba04f342bfe8129a7f5222" src="https://user-images.githubusercontent.com/8672791/97790364-79743480-1bc8-11eb-914a-d35a4702e1ab.png"> - after: <img width="300" alt="now" src="https://user-images.githubusercontent.com/8672791/97790374-842ec980-1bc8-11eb-8890-9b0164ceeca5.png">

**Comment by cart** [comment]
Arg nice catch. We should definitely leave this open.

**Comment by memoryruins** [comment]
Resolved by #765
